pyidr/file_pattern.py

```diff
@@ -57,10 +61,10 @@ def expand_range(r):
             raise ValueError("inverted range: %s" % r)
         step = int(step)
         if len(start_str) != len(stop_str):
-            return map(str, range(start, stop, step))
+            return list(map(str, list(range(start, stop, step))))
```

Comment:
That inner `list` really is necessary? Or could even get rid of both with a list comprehension.

Reply:
See f400c6f